app/controllers/user_controller.rb

```diff
@@ -1,2 +1,12 @@
 class UserController < ApplicationController
```

Comment:
It's not `UserController` but **`UsersController`**. You've forgotten the `s` 
You have the same problem with the filename

Reply:
This model was generated by the devise gem.